src/cpp/src/visual_language/qwen2_5_vl/classes.cpp

```diff
@@ -0,0 +1,225 @@
+// Copyright (C) 2023-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "visual_language/qwen2_5_vl/classes.hpp"
+
+#include "utils.hpp"
+
+namespace ov::genai {
+
+InputsEmbedderQwen2_5_VL::InputsEmbedderQwen2_5_VL(
+    const VLMConfig& vlm_config,
+    const std::filesystem::path& model_dir,
+    const std::string& device,
+    const ov::AnyMap device_config) :
+    InputsEmbedderQwen2VL(vlm_config, model_dir, device, device_config) {}
+
+InputsEmbedderQwen2_5_VL::InputsEmbedderQwen2_5_VL(
+    const VLMConfig& vlm_config,
+    const ModelsMap& models_map,
+    const Tokenizer& tokenizer, 
+    const std::filesystem::path& config_dir_path,
+    const std::string& device,
+    const ov::AnyMap device_config) :
+    InputsEmbedderQwen2VL(vlm_config, models_map, tokenizer, config_dir_path, device, device_config) {}
+
+ov::Tensor InputsEmbedderQwen2_5_VL::run_image_embeddings_merger(
+    const std::vector<EncodedImage>& images, 
+    const std::vector<size_t>& images_sequence, 
+    size_t image_id, 
+    const VLMConfig& vlm_config) {
```

Comment:
VLMConfig isn't used by both Qwens. I guess we missed it while factoring out. Remove it please

Reply:
Removed